README.md

```diff
@@ -45,8 +45,14 @@ This states that `process.env.NODE_ENV` must be set to `development`,
 A default value may be provided:
 
 ```javascript
-const NODE_ENV = envvar.enum('NODE_ENV', ['development', 'staging', 'production'], 'production');
+const NODE_ENV = envvar.oneOf('NODE_ENV', ['development', 'staging', 'production'], 'production');
 ```
 
 This states that `process.env.NODE_ENV` must either be unset (in which case the
 default value is assumed), or set to `development`, `staging`, or `production`.
+
+
+### Migration from `1.x` to `2.x`
+
+  * `enum` was renamed to `oneOf` to avoid shadowing the reserved typescript
+    keyword with the same name.
```

Comment:
That is fine by me. I have seen both practices in the past. Given that we are not making a lot of changes to this package, do you think people are likely to look at the release notes when they decide to do the major version bump? Maybe we should include a link to the releases page in the readme if that is where we are going to track changes.